docs/en/advanced_guides/datasets/scannet_det.md

```diff
@@ -226,21 +226,14 @@ scannet
 - `posed_images/scenexxxx_xx`: The set of `.jpg` images with `.txt` 4x4 poses and the single `.txt` file with camera intrinsic matrix.
 - `scannet_infos_train.pkl`: The train data infos, the detailed info of each scan is as follows:
-  - info\['point_cloud'\]: {'num_features': 6, 'lidar_idx': sample_idx}.
-  - info\['pts_path'\]: The path of `points/xxxxx.bin`.
-  - info\['pts_instance_mask_path'\]: The path of `instance_mask/xxxxx.bin`.
-  - info\['pts_semantic_mask_path'\]: The path of `semantic_mask/xxxxx.bin`.
-  - info\['annos'\]: The annotations of each scan.
-    - annotations\['gt_num'\]: The number of ground truths.
-    - annotations\['name'\]： The semantic name of all ground truths, e.g. `chair`.
-    - annotations\['location'\]: The gravity center of the axis-aligned 3D bounding boxes in depth coordinate system. Shape: \[K, 3\], K is the number of ground truths.
-    - annotations\['dimensions'\]: The dimensions of the axis-aligned 3D bounding boxes in depth coordinate system, i.e. (x_size, y_size, z_size), shape: \[K, 3\].
-    - annotations\['gt_boxes_upright_depth'\]: The axis-aligned 3D bounding boxes in depth coordinate system, each bounding box is (x, y, z, x_size, y_size, z_size), shape: \[K, 6\].
-    - annotations\['unaligned_location'\]: The gravity center of the axis-unaligned 3D bounding boxes in depth coordinate system.
-    - annotations\['unaligned_dimensions'\]: The dimensions of the axis-unaligned 3D bounding boxes in depth coordinate system.
-    - annotations\['unaligned_gt_boxes_upright_depth'\]: The axis-unaligned 3D bounding boxes in depth coordinate system.
-    - annotations\['index'\]: The index of all ground truths, i.e. \[0, K).
-    - annotations\['class'\]: The train class ID of the bounding boxes, value range: \[0, 18), shape: \[K, \].
+  - info\['lidar_points'\]\['lidar_path'\]: The filename of `xxx.bin` of lidar points.
+  - info\['lidar_points'\]\['num_pts_feats'\]: The feature dimension of point.
+  - info\['lidar_points'\]\['pts_semantic_mask_path'\]: The filename of `xxx.bin` contains semantic mask annotation.
+  - info\['lidar_points'\]\['pts_instance_mask_path'\]: The filename of `xxx.bin` contains semantic mask annotation.
+  - info\['lidar_points'\]\['axis_align_matrix'\]: The transformation matrix to align the axis.
+  - info\['instances'\]: A list of dict contains all annotations, each dict contains all annotation information of single instance.
+    - info\['instances'\]\[i\]\['bbox_3d'\]: The axis-aligned 3D bounding boxes in depth coordinate system, each bounding box is (x, y, z, x_size, y_size, z_size), shape: \[K, 6\].
+    - info\['instances'\]\[i\]\['bbox_label_3d'\]: The label of each 3d bounding boxes.
 - `scannet_infos_val.pkl`: The val data infos, which shares the same format as `scannet_infos_train.pkl`.
 - `scannet_infos_test.pkl`: The test data infos, which almost shares the same format as `scannet_infos_train.pkl` except for the lack of annotation.
 
```

Comment:
double-check the train_pipieline in the following code, especailly 'PointSegClassMapping' parameter